src/components/NavBar/SearchBar.tsx

```diff
@@ -163,11 +163,11 @@ export const SearchBar = () => {
             !isOpen && !isMobile && magicalGradientOnHover,
             isMobileOrTablet && (isOpen ? styles.visible : styles.hidden)
           )}
-          borderRadius={isOpen || isMobileOrTablet ? undefined : '12'}
-          borderTopRightRadius={isOpen && !isMobile ? '12' : undefined}
-          borderTopLeftRadius={isOpen && !isMobile ? '12' : undefined}
+          borderRadius={isOpen || isMobileOrTablet ? undefined : '16'}
+          borderTopRightRadius={isOpen && !isMobile ? '16' : undefined}
+          borderTopLeftRadius={isOpen && !isMobile ? '16' : undefined}
           borderBottomWidth={isOpen || isMobileOrTablet ? '0px' : '1px'}
```

Comment:
border is missing on the bottom left and right corners when open

before:
<img width="548" alt="Screenshot 2023-09-12 at 9 44 33 AM" src="https://github.com/Uniswap/interface/assets/11512321/3eac6eef-6019-4dbb-a36b-3527f1a587bc">

after:
![Screenshot 2023-09-12 at 9 38 34 AM](https://github.com/Uniswap/interface/assets/11512321/1b75a101-4233-4548-baf0-c22d456d2dbd)

Reply:
Fix'd